app/src/main/res/layout/item_top_album.xml

```diff
@@ -0,0 +1,36 @@
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
WARNING: [UnusedResources] The resource `R.layout.item_top_album` appears to be unused
 Unused resources make applications larger and slow down builds.